### PR TITLE
6 get api articles article id comments

### PR DESCRIPTION
**Pull request by avgrimshaw**

### CODE REVIEW

File: __tests__/articles.test.js [hunk 1]
```diff
@@ -81,3 +81,44 @@ describe("GET /api/articles/:article_id", () => {
...
+            comment_id: expect.any(Number),
+            body: expect.any(String),
+            article_id: expect.any(Number),
+            author: expect.any(String),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+          });
+        });
+      });
+  });
```
Comment (nickkagame) [inline __tests__/articles.test.js:108]: One missing test and behaviour - Status 200, valid ID, but has no comments responds with an empty array of comments.  
You will need to check the reason for the query returning empty rows:
article exists but has no comments.
article does not exist.
You will need to check whether the article exists in order to determine this.

**Review by nickkagame — CHANGES_REQUESTED** [review]
Hi Adam, Good start with this one.  There's one key behaviour missing. See my comment for details!  (Think back to Stephen's lecture on Tuesday)

**Review by johnhutcheon — APPROVED** [review]
Thanks for making the change Adam. good to merge 👍
